static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx

```diff
@@ -70,6 +71,10 @@ import {SamplingSDKClientRateChangeAlert} from './samplingSDKClientRateChangeAle
 import {SamplingSDKUpgradesAlert} from './samplingSDKUpgradesAlert';
 import {isUniformRule, SERVER_SIDE_SAMPLING_DOC_LINK} from './utils';
 
+const LimitedAvailabilityProgramEndingAlert = HookOrDefault({
+  hookName: 'component:dynamic-sampling-limited-availability-program-ending',
+});
```

Comment:
Why use HookOrDefault if there’s no default? Why not just <Hook />

Reply:
https://github.com/getsentry/sentry/pull/39458#discussion_r985017228